.travis.yml

```diff
@@ -13,6 +13,7 @@ sudo: false
 
 env:
   - TEST_TARGET=default
+  - TEST_TARGET=default TEST_MINIMAL=true
   - TEST_TARGET=coding
```

Comment:
Good idea. I'd suggest it as a follow-up activity as it doesn't look like it makes a huge difference right now, e.g. 2 mins out of 15-21 mins.